tests/ha/test_pod_failure.py

```diff
@@ -2106,7 +1898,7 @@ def test_mpu_during_pod_unsafe_shutdown(self):
         LOGGER.info("Step 2: Shutdown the data pod by deleting deployment (unsafe)")
         LOGGER.info("Get pod name to be deleted")
         pod_list = self.node_master_list[0].get_all_pods(pod_prefix=const.POD_NAME_PREFIX)
-        pod_name = random.sample(pod_list, 1)[0]
+        pod_name = self.system_random.sample(pod_list, 1)[0]
```

Comment:
Why pod deletion is not optimized in this test?

Reply:
There is one more PR i will be raising for during tests